eegnb/cli/introprompt.py

```diff
@@ -40,57 +40,61 @@ def intro_prompt():
           f"[10] {boards[10]} \n",
           f"[11] {boards[11]} \n")
 
-    board_idx = int(input('Enter Board Selection:'))
+    board_idx = int(input('Enter Board Selection: '))
     board_selection = board_codes[board_idx]    # Board_codes are the actual names to be passed to the EEG class
     print(f"Selected board {boards[board_idx]} \n")
 
     # Handles wifi shield connectivity selection if an OpenBCI board is being used
     if board_selection in ['cyton', 'cyton_daisy', 'ganglion']:
-        # if the ganglion is being used, will also need the MAC address
-        if board_selection == 'ganglion':
-            print("Please enter the Ganglions MAC address:\n")
-            mac_address = input("MAC address:")
 
         # determine whether board is connected via Wifi or BLE
         print("Please select your connection method:\n"
               "[0] usb dongle \n"
               "[1] wifi shield \n")
-        connect_idx = input("Enter connection method:")
+        connect_idx = int(input("Enter connection method: "))
 
         # add "_wifi" suffix to the end of the board name for brainflow
         if connect_idx == 1:
             board_selection = board_selection + "_wifi"
+            ip_address = input("\nEnter Ganglion+WiFi IP Address: ")
```

Comment:
Ganglion is not the only device that can be used with the WiFi shield.

```suggestion
            ip_address = input("\nEnter IP Address of device: ")
```

Reply:
This prompt did not exist before....

Reply:
Hey @ErikBjare although you technically can use the Cyton/daisy with the Wifi shield, it is not recommended by OpenBCI due to some issues with packet loss so we made the decision to only officially support it with the Ganglion in the library.

Reply:
I knew about the OpenBCI warning, but didn't know about the decision to not even attempt to support it here. 

Makes sense, although wouldn't the flow as it looks right now let you choose Wi-Fi as a connection option for other OpenBCI devices?

Reply:
@ErikBjare Good point. I was on the fence on whether or not to include Cyton+Wifi, but I can apply @JadinTredup's conclusion now. Editing now to enforce this.

Reply:
Maybe add a code comment while you're at it :)

Reply:
Here is the revised code block that handles all supported OpenBCI Boards via CLI w/ input prompts:

```
    # Handles connectivity selection if an OpenBCI board is being used
    if board_selection in ['cyton', 'cyton_daisy', 'ganglion']:

        # determine whether board is connected via Wifi or BLE
        print("Please select your connection method:\n"
              "[0] usb dongle \n"
              "[1] wifi shield \n")
        connect_idx = int(input("Enter connection method: "))

        # add "_wifi" suffix to the end of the board name for brainflow
        if connect_idx == 1:
            board_selection = board_selection + "_wifi"

        if board_selection == 'ganglion':
            # If the ganglion is being used, you can enter optional Ganglion mac address
            ganglion_mac_address = input("\nGanglion MAC Address (Press Enter to Autoscan): ")    
        elif board_selection == 'ganglion_wifi':
            # IP address is required 
            ip_address = input("\nEnter Ganglion+WiFi IP Address: ")
        elif board_selection == 'cyton_wifi' or board_selection == 'cyton_daisy_wifi':
            print(f"\n{boards[board_idx]} + WiFi is not supported. Please use the dongle that was shipped with the device.\n")
            exit()
```


Full Console Log:

```
(eeg-nb-main) Richard-Waltmans-iMac-4:eeg-notebooks Richard$ eegnb runexp -ip
run command line prompt script
Welcome to NeurotechX EEG Notebooks. 
Please enter the integer value corresponding to your EEG device: 
[0] None 
[1] Muse2016 
[2] Muse2 
[3] MuseS 
[4] OpenBCI Ganglion 
[5] OpenBCI Cyton 
[6] OpenBCI Cyton + Daisy 
[7] G.Tec Unicorn 
[8] BrainBit 
[9] Notion 1 
[10] Notion 2 
 [11] Synthetic 

Enter Board Selection: 6
Selected board OpenBCI Cyton + Daisy 

Please select your connection method:
[0] usb dongle 
[1] wifi shield 

Enter connection method: 1

OpenBCI Cyton + Daisy + WiFi is not supported. Please use the dongle that was shipped with the device.
```